src/pages/iv-items/forms/ItemsForm.js

```diff
@@ -0,0 +1,575 @@
+
+    validationSchema: yup.object({
+      categoryId: yup.string().required(),
+      sku: yup.string().required(),
+      name: yup.string().required(),
+      priceType: yup.string().required(),
+      msId: yup.string().required(),
+      lotCategoryId: yup
+        .string()
+        .nullable()
+        .test(function (value) {
+          const { trackBy } = this.parent
+
+          return trackBy === '2' || trackBy === 2 ? value != null && value.trim() !== '' : true
+        }),
+      spfId: yup
+        .string()
+        .nullable()
+        .test(function (value) {
+          const { trackBy } = this.parent
+
+          return trackBy === '1' || trackBy === 1 ? value != null && value.trim() !== '' : true
+        })
+    }),
+    onSubmit: async obj => {
+      const recordId = obj.recordId
+
+      try {
+        const response = await postRequest({
+          extension: InventoryRepository.Items.set,
+          record: JSON.stringify({ ...obj, attachment: null })
+        })
+
+        if (!recordId) {
+          toast.success(platformLabels.Added)
+          formik.setValues({
+            ...obj,
+            recordId: response.recordId
+          })
+          setStore(prevStore => ({
+            ...prevStore,
+            recordId: response.recordId,
+            _msId: formik.values.msId
+          }))
+        } else {
+          toast.success(platformLabels.Edited)
+        }
+        setFormikInitial(formik.values)
+
+        if (imageUploadRef.current) {
+          await imageUploadRef.current.submit()
+        }
+
+        const res = await getRequest({
+          extension: InventoryRepository.Items.get,
+          parameters: `_recordId=${response.recordId}`
+        })
+
+        formik.setFieldValue('sku', res.record.sku)
+      } catch (error) {}
+
+      invalidate()
+    }
+  })
+
+  const editMode = !!recordId || formik.values.recordId
+
+  const { changeDT, maxAccess } = useRefBehavior({
+    access: access,
+    readOnlyOnEditMode: editMode
+  })
+
+  useEffect(() => {
+    ;(async function () {
+      try {
+        if (recordId) {
+          const res = await getRequest({
+            extension: InventoryRepository.Items.get,
+            parameters: `_recordId=${recordId}`
+          })
+          setFormikInitial(res.record)
+          formik.setValues({ ...res.record, kitItem: !!res.record.kitItem })
+          setShowLotCategories(res.record.trackBy === '2' || res.record.trackBy === 2)
+          setShowSerialProfiles(res.record.trackBy === '1' || res.record.trackBy === 1)
+          setStore(prevStore => ({
+            ...prevStore,
+            _msId: res.record.msId,
+            measurementId: res.record.defSaleMUId,
+            priceGroupId: res.record.pgId,
+            returnPolicy: res.record.returnPolicyId
+          }))
+        }
+      } catch {}
+    })()
+  }, [])
+
+  return (
+    <FormShell resourceId={ResourceIds.Items} form={formik} maxAccess={maxAccess} editMode={editMode}>
```

Comment:
in the jira ticket Nora mentioned there two buttons:  record remarks and integration accounts
and they are not rendered here don't we need them?